python/lsst/afw/geom/ellipses/__init__.py

```diff
@@ -23,23 +23,20 @@
 """lsst.afw.geom.ellipses
 """
 from __future__ import absolute_import
-from builtins import object
 from .ellipsesLib import *
```

Comment:
@TallJimbo you are absolutely right. Here. There are many other instances that are not in `__init__.py` files and it was my mistake moving this comment too far up to an `__init__.py` file. My apologies.